src/content/get-started/ch2-add-commands.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Making your CLI program interactive
+description: Add simple commands to your cli program. Learn the fundamentals of Dart syntax including control flow, collections, variables, functions, and more.
+---
+
+In this chapter, you'll get hands-on practice with Dart syntax. You'll learn how
+to read user input, print usage information, and create a basic command-line
+interaction. By the end, you'll have a foundational understanding of Dart
+syntax.
+
+:::secondary What you'll learn
+
+* Implement basic control flow with `if/else`.
+* Work with collections, specifically `List`s, and perform common operations
+  like checking if a list is empty.
+* Declare and use variables with `const` and `late String?`.
+* Handle nullability with null checks.
+* Define and call functions.
+* Use string interpolation for dynamic text.
+* Read user input from the command line using `stdin`.
+
+:::
+
+## Prerequisites
+
+Before you begin this chapter, ensure you have:
+
+* Completed Chapter 1 and have a working Dart development environment.
+* Familiarity with basic programming concepts (variables, data types, control
+  flow).
+
+## Tasks
+
+Start building your **Dartpedia** command-line application by adding some basic
+functionality and exploring Dart syntax.
+
+### Task 1: Implement version and help commands
+
+1.  **Implement the `version` command in `cli/bin/cli.dart`:** Add logic to
+    handle a `version` command. Use an `if` statement to check if the first
+    argument provided is `version`. You'll also need a `version` constant.
+
+    First, above your `main` function, declare a `const` variable for the
+    version:
+
+    ```dart
+    const version = '0.0.1'; // Add this line
```

Comment:
I think there should be 1 sentence about what `const` means in the simplest terms possible (i.e. "it can never be changed")